exercises/practice/largest-series-product/Example.cs

```diff
@@ -0,0 +1,34 @@
+using System;
```

Comment:
Should that file be renamed without change instead of being deleted then added?

The new file is the same as the old file.

New file:
https://github.com/ErikSchierboom/csharp/blob/v3-migration/exercises/practice/largest-series-product/Example.cs

Old file:
https://github.com/exercism/csharp/blob/main/exercises/largest-series-product/Example.cs

Reply:
No idea why that is. Git and renames can sometimes be weird. I think it's fine to merge in the grand scheme of things. Thanks for the review!